Changelog.md

```diff
@@ -8,6 +8,7 @@ Important Bugfixes:
 Language Features:
  * ABIEncoderV2: Implement encoding of calldata arrays and structs.
  * Code Generation: Implement copying recursive structs from storage to memory.
+ * Static Analyzer: Disallow libraries calling themselves externally.
```

Comment:
Please move.

Reply:
Moved it to the bugfix section as assumed that's what you meant ;)

Reply:
Ah sorry, - no I meant rebase onto develop and move it to the new release that will then magically appear.

Reply:
Ok!